### ISSUE TITLE
ImportError: cannot import name 'LocallyConnected2D' from 'tensorflow.keras.layers'

### ISSUE THREAD

**Issue by sanket-tss** [serengil/deepface#1094]
Got this error while building with below requirements

fastapi==0.109.0
uvicorn==0.27.0
Pillow==10.0.1
python-multipart==0.0.6
deepface==0.0.83

```python
2024-03-10 15:13:34   File "/app/app.py", line 2, in <module>
2024-03-10 15:13:34     from deepface import DeepFace
2024-03-10 15:13:34   File "/usr/local/lib/python3.10/site-packages/deepface/DeepFace.py", line 15, in <module>
2024-03-10 15:13:34     from deepface.modules import (
2024-03-10 15:13:34   File "/usr/local/lib/python3.10/site-packages/deepface/modules/modeling.py", line 5, in <module>
2024-03-10 15:13:34     from deepface.basemodels import VGGFace, OpenFace, FbDeepFace, DeepID, ArcFace, SFace, Dlib, Facenet
2024-03-10 15:13:34   File "/usr/local/lib/python3.10/site-packages/deepface/basemodels/FbDeepFace.py", line 29, in <module>
2024-03-10 15:13:34     from tensorflow.keras.layers import (
2024-03-10 15:13:34 ImportError: cannot import name 'LocallyConnected2D' from 'tensorflow.keras.layers' (/usr/local/lib/python3.10/site-packages/keras/_tf_keras/keras/layers/__init__.py)
```

**Comment by freedom9393** [comment]
Already posted [here](https://github.com/serengil/deepface/issues/1084)

**Comment by serengil** [comment]
TLDR; sorted in v0.0.86
